lightkurve/periodogram.py

```diff
@@ -2,64 +2,251 @@
 from astropy import units as u
 from astropy.stats import LombScargle
 from matplotlib import pyplot as plt
+from scipy.ndimage.filters import gaussian_filter
+from astropy.units import cds
+from astropy.table import Table
+import astropy
 
-__all__ = ['Periodogram']
+import logging
+from . import PACKAGEDIR, MPLSTYLE
 
 
+log = logging.getLogger(__name__)
+
+__all__ = ['Periodogram', 'estimate_mass', 'estimate_radius',
+            'estimate_mean_density', 'stellar_params', 'standardize_units']
+
+###Add uncertainties <- review these [astero]
+numax_s = 3090.0 # Huber et al 2013
+deltanu_s = 135.1
+teff_s = 5777.0
+
+class InputError(Exception):
+    """Raised if user inputs both frequency and period kwargs"""
+    pass
```

Comment:
A common convention in Python is to raise a `ValueError` if there is a problem with the values of the input.  I would recommend to raise a `ValueError` here too, for the sake of not requiring the user to have to import a custom exception.  (The `ValueError` is in the standard library and does not need to be imported -- easier to use!)

Reply:
I have replaced all instances of InputError with ValueErrors!